src/box/lua/schema.lua

```diff
@@ -259,7 +261,8 @@ end
  @example
  check_param_table(options, { user = 'string',
                               port = 'string, number',
-                              data = 'any'} )
+                              data = 'any',
+                              addr = function(opt) ... end} )
```

Comment:
Nit: Let's add a function body to make this example more useful.

Reply:
Done